articles/compliance/definitions.md

```diff
@@ -0,0 +1,18 @@
+---
+title: General Data Protection Regulation (GDPR) Definitions
+description: Definitions used for Auth0's documentation on GDPR
+---
+# Definitions
+
+| Term | Definition |
+| - | - |
+| Subject | An individual/natural person |
+| Data Controller | The entity that collects and processes data on subjects (see GDPR for exact definition) |
```

Comment:
For convenience, could it be possible to add a link to the official resources where this definition can be found?